nflows/transforms/splines/rational_quadratic.py

```diff
@@ -139,6 +139,10 @@ def rational_quadratic_spline(
         c = -input_delta * (inputs - input_cumheights)
 
         discriminant = b.pow(2) - 4 * a * c
+
+        float_precision_mask = (torch.abs(discriminant)/(b.pow(2) + 1e-8)) < 1e-6
+        discriminant[float_precision_mask] = 0
```

Comment:
Maybe use `torch.where` here, just to be safe?

Reply:
Indeed, it is a much better operation in this case. I have corrected accordingly!